gazebo_ros/src/executor.cpp

```diff
@@ -41,7 +48,9 @@ void Executor::run()
 
 void Executor::shutdown()
 {
-  rclcpp::shutdown();
```

Comment:
I suspect the issue is that we're no longer properly shutting down the context (only the executor itself). I can take a closer look.

Reply:
Reverting this PR didn't seem to have any positive effect: https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1227

Reply:
> Reverting this PR didn't seem to have any positive effect: #1227

I see the revert was finally closed, is the problem fixed?
I can take another look to this if not

Reply:
it was fixed by #1228